tests/javascript/newsletter_editor/blocks/automatedLatestContentLayout.spec.js

```diff
@@ -567,14 +567,6 @@ describe('Automated latest content', function () {
           });
         });
       });
-
-      it.skip('closes the sidepanel after "Done" is clicked', function () {
-        var mock = sinon.mock().once();
-        global.MailPoet.Modal.cancel = mock;
-        view.$('.mailpoet_done_editing').click();
-        mock.verify();
-        delete (global.MailPoet.Modal.cancel);
-      });
```

Comment:
Why was this removed?

Reply:
Well, I renamed the file. I could've copied it, but I felt like there is no reason to keep the test around. We keep the file automatedLatestContent.js just for back-compatibility purposes. We don't update it with new features, we don't fix bugs in it, so I felt like there is no need for the test.